### PR TITLE
Swagger based node SDK for Dns

### PR DESCRIPTION
**Pull request by karravi-msft**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@karravi-msft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: lib/services/dnsManagement2/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "contributors": [    
+    "Block, Glenn <gblock@microsoft.com>",    
+    "Cowlishaw, Mark <markcowl@microsoft.com>",    
+    "Dejardin, Louis <loudej@microsoft.com>",    
+    "Georgiev, Yavor <yavorg@microsoft.com>",    
+    "Janczuk, Tomasz <tjanczuk@microsoft.com>",    
+    "Rodrigues, Andre <andrerod@microsoft.com>",    
+    "Tavares, Chris <ctavares@microsoft.com>"    
+  ],    
+  "version": "0.11.1",
```
Comment (amarzavery) [inline lib/services/dnsManagement2/package.json]: 0.1.0

File: lib/services/dnsManagement2/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{    
+  "name": "azure-arm-dns",    
+  "author": "Microsoft Corporation",    
+  "contributors": [    
+    "Block, Glenn <gblock@microsoft.com>",
```
Comment (amarzavery) [inline lib/services/dnsManagement2/package.json]: Remove all these names and add your name

File: lib/services/dnsManagement2/package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  "tags": [    
+    "azure",    
+    "sdk"    
+  ],    
+  "keywords": [    
+    "node",    
+    "azure"    
+  ],    
+  "main": "./lib/dnsManagementClient.js",    
+  "engines": {
```
Comment (amarzavery) [inline lib/services/dnsManagement2/package.json]: remove the engines property

File: lib/services/dnsManagement2/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Microsoft Azure SDK for Node.js - DNS Management    
+    
+This project provides a Node.js package that makes it easy to manage Microsoft Azure DNS.     
+    
+## How to Install    
+    
+```bash    
+npm install azure-arm-dns    
+```
```
Comment (amarzavery) [inline lib/services/dnsManagement2/README.md]: Please provide samples on CRUD. Please take a look at [README of storagemanagement](https://github.com/Azure/azure-sdk-for-node/blob/master/lib/services/storageManagement2/README.md#how-to-use).

**Comment by amarzavery** [comment]
@karravi-msft - As  you can see, the arm version of dnsmanagement was already published. For services like compute - computeManagement represents rdfe based client and computeManagement2 represents arm based client. However, dns never had an rdfe client. We do not need dnsManagement2 folder. So, please update the code in dnsManagement itself.

**Comment by amarzavery** [comment]
https://github.com/Azure/azure-sdk-for-node/blob/master/lib/services/dnsManagement/lib/dnsManagementClient.js#L115

File: lib/services/dnsManagement2/package.json (hunk 2 above)
Comment (karravi-msft) [inline lib/services/dnsManagement2/package.json]: Done

File: lib/services/dnsManagement2/package.json (hunk 3 above)
Comment (karravi-msft) [inline lib/services/dnsManagement2/package.json]: Done

File: lib/services/dnsManagement2/README.md (hunk 4 above)
Comment (karravi-msft) [inline lib/services/dnsManagement2/README.md]: will add this later.. currently not changing this since it is the same for prev release as well

**Comment by amarzavery** [comment]
@karravi-msft - Can you comment out lines 1125 and 1126 in `lib/azure.js` for now.
```
//var azureARMDns = require('azure-arm-dns');
//exports.ARMDnsManagementClient = azureARMDns.DnsManagementClient;
```
I shall fix this soon.

**Comment by amarzavery** [comment]
sorry one more line 1138.

**Pull request merged**